### PR TITLE
Development

### PR DESCRIPTION
**Pull request by gferrarocamus**
Code for "Project: Embedding Images and Video"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,150 @@
 <html>
```
Comment (inhaq) [inline index.html]: Add `<!DOCTYPE html>` at the top of the page in order to instruct the web browser about what version of HTML.
Add a `lang` attribute to the `html` tag to declare the language of this document.

File: index.html [hunk 2]
```diff
@@ -1,11 +1,150 @@
...
+        </form>
+      </div>
+    </nav>
+
+    <!-- CONTENT -->
+
+    <div class="row" id="page-content">
+      <!-- MAIN VIDEO -->
+      <div id="video" class="col-md-8 mx-auto">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0"
```
Comment (inhaq) [inline index.html]: `frameborder="0"` is obsolete. Kindly remove it.

File: index.html [hunk 3]
```diff
@@ -1,11 +1,150 @@
...
+        <span>Video</span>
+      </div>
+
+      <!-- COMMENTS -->
+      <div id="comments" class="col-md-8">
+        <div class="container comment">
+          <div class="row">
+            <div class="col-2">
+              <img class="rounded-circle"
+                src="https://yt3.ggpht.com/-tVdji_W8sak/AAAAAAAAAAI/AAAAAAAAAAA/mFtCyL-VwY0/s48-c-k-no-mo-rj-c0xffffff/photo.jpg">
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.

File: index.html [hunk 4]
```diff
@@ -1,11 +1,150 @@
...
+              <p class="comment-text">"use your powers for good" haha
+                thank you for the video.</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="container comment">
+          <div class="row">
+            <div class="col-2">
+              <img class="rounded-circle"
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.

File: index.html [hunk 5]
```diff
@@ -1,11 +1,150 @@
...
+              <p class="username"><strong>Sibe Cute</strong></p>
+              <p class="comment-text">thanks a lot for video</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="container comment">
+          <div class="row">
+            <div class="col-2">
+              <img class="rounded-circle"
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.

File: index.html [hunk 6]
```diff
@@ -1,11 +1,150 @@
...
+              <p class="username"><strong>ButtonMasher</strong></p>
+              <p class="comment-text">Wow this video was super helpful, clear, and concise. Thank you very much.﻿</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="container comment">
+          <div class="row">
+            <div class="col-2">
+              <img class="rounded-circle"
```
Comment (inhaq) [inline index.html]: An `img` element must have an `alt` attribute that is to be rendered when the element to which it is applied cannot be rendered.

File: index.html [hunk 7]
```diff
@@ -1,11 +1,150 @@
...
+              <p class="comment-text">Nice.Thanks a lot :)﻿</p>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
```
Comment (inhaq) [inline index.html]: Kindly remove these libraries for first few projects in order to learn more about CSS 😊

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+iframe {
+    width: 100%
+}
+
+#videoTitle h3 {
+  font-size: 24px;
+  padding-top:20px;
+}
+
+#videoTitle, #channel {
```
Comment (inhaq) [inline styles.css]: Consider looking into [this resource](https://css-tricks.com/the-difference-between-id-and-class/) in order to learn more about CSS class and id. Kindly figure out what you are doing wrong here.
Hint: Problem is with selectors on line 9 and 14.

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  font-size: 20px;
+  padding-top:20px;
+}
+
+#sidebar img {
+  width: 50%;
+  padding: 10px;
+}
+
+#comments {
```
Comment (inhaq) [inline styles.css]: I would suggest using mostly classes instead of using id most time.

File: index.html [hunk 10]
```diff
@@ -1,11 +1,150 @@
...
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
   <title>Embedding Images and Video</title>
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
```
Comment (inhaq) [inline index.html]: Kindly don't use Bootstrap unless you're told to use it in the project.

File: index.html [hunk 11]
```diff
@@ -1,11 +1,150 @@
...
+    integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous" />
+  <link rel="stylesheet" href="styles.css" />
 </head>
+
 <body>
-  
+  <div class="container">
+    <!-- NAVBAR -->
```
Comment (inhaq) [inline index.html]: Good to add comments in code 👍🏻

File: index.html [hunk 12]
```diff
@@ -1,11 +1,150 @@
...
+    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-light">
```
Comment (inhaq) [inline index.html]: Kindly write your own code for `nav` instead of using bootstrap.

File: styles.css (hunk 8 above)
Comment (1ba1) [reply styles.css]: Hi, thanks for the review.
We looked at the resource you linked, but couldn't be able to figure out what is wrong with our code.
Are you suggesting to use a single class titles for both video and channel titles?
Thanks in advance.

Comment (inhaq) [reply styles.css]: Hi @1ba1, `id` is something you'll need to use only once in your CSS document whereas `classes` can be used as much time you needed. Here on line 9 and 14, you're using `id` twice in CSS document, use `class` instead.

Comment (1ba1) [reply styles.css]: ok thanks!

**Comment by gferrarocamus** [comment]
Thanks for the review, @inhaq - we've just resubmitted the project.

**Comment by inhaq** [comment]
### Great job 🥇on changes
It's 👍🏻 from my side and you can merge this branch as well as mark this project completed by submitting project completion form.
Suggestion: You can work more on the design part of this project to make it look more like YouTube. [This](https://snag.gy/QmOj1r.jpg) is how it looks with me. No need to resubmit this project for review. Congrats 🎉

**Pull request merged**